src/fromager/vendor_rust.py

```diff
@@ -117,6 +117,31 @@ def _detect_rust_build_backend(
     return None
 
 
+def vendor_generic_rust_package(
+    req: Requirement,
+    manifests: list[pathlib.Path],
+    root_dir: pathlib.Path,
+    *,
+    shrink_vendored: bool = True,
+) -> typing.Iterable[pathlib.Path]:
+    """Vendor crates of a generic Rust package"""
+    if not manifests:
+        # default to Cargo.toml in root dir
+        manifests = [root_dir / "Cargo.toml"]
+    # fetch and vendor Rust crates
+    vendored = _cargo_vendor(req, manifests, root_dir)
+    logger.debug(f"{req.name}: vendored crates: {sorted(d.name for d in vendored)}")
```

Comment:
Is there a reason why we would want to sort vendored crates?

Reply:
I like to sort items for logging to have a consistent output. The file system can return files in arbitrary order.

Reply:
Ah, I see!
Thank you for the clarification